alacritty/src/display/hint.rs

```diff
@@ -225,6 +226,13 @@ impl HintMatch {
         self.hyperlink.as_ref()
     }
 
+    pub fn get_hint_mouse_button(&self) -> MouseButton {
```

Comment:
```suggestion
    pub fn mouse_button(&self) -> MouseButton {
```

Would probably also make sense to add a doc comment that explains that it will default to `Left`.